lib/net/nforceif/src/sys_arch.c

```diff
@@ -118,7 +117,7 @@ sys_thread_new(const char *name, lwip_thread_fn function, void *arg, int stacksi
     abort();
   }
 
-  st->handle = XCreateThread((void *)function, arg, NULL);
+  st->handle = CreateThread(NULL, stacksize, (void *)function, arg, 0, NULL);
```

Comment:
Considering this is a driver, I wonder if we should use kernel functions directly instead.
I'm not sure if this might affect things like callbacks (done on behalf of these thread) in which people might query thread information.

Reply:
I did consider that, but I'm not sure whether that thread may touch TLS-stuff (like `errno`, which became thread local in upstream PDCLib), so I decided to go the safer route.

Reply:
Fair enough; ideally none of this should touch `errno`. We might want to look for a better solution in the future.